otel-integration/k8s-helm/values.yaml

```diff
@@ -121,6 +121,10 @@ opentelemetry-agent:
       enabled: true
     hostMetrics:
       enabled: true
+      # Enables process metrics scraping.
+      # Disabled by default, requires privilleged mode
+      process:
```

Comment:
Should we add the preset also to Windows and other values files?

Reply:
According to docs Windows should be supported in `process` scraper: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/README.md#getting-started

Either way I would suggest either adding it or document it's not supported or something. I'll leave it up to you. The PR looks good otherwise 👍